src/main/azure/proxies.json

```diff
@@ -14,6 +14,20 @@
                 "route": "/api/question"
             },
             "backendUri": "https://localhost/api/createQuestion"
+        },
+        "proxy3": {
+            "matchCondition": {
+                "methods": [ "GET" ],
+                "route": "/api/topics"
+            },
+            "backendUri": "https://localhost/api/getTopic"
+        },
+        "proxy4": {
+            "matchCondition": {
+                "methods": [ "POST" ],
```

Comment:
Given that we now express the route within the function itself (much neater, really like it :) ) could we move these into their respective functions?

Reply:
done